src/github-api.spec.ts

```diff
@@ -0,0 +1,80 @@
+jest.mock("../src/github-api");
+jest.mock("./fetch");
+
+const MockedGithubAPI = require("./github-api").default;
```

Comment:
if this is using the mocked version, then does it really make sense to test it? 🤔

Reply:
Indeed, it sounds odd. I added two commits to switch to the real github-api import.

The CI failure on the first commit shows why a mocked version was used: to go around the need of having a GITHUB_AUTH env var (https://github.com/lerna/lerna-changelog/blob/v1.0.1/src/__mocks__/github-api.ts#L3-L7). In a second commit, I faked the env var. It probably looks better. 🤔